### PR TITLE
benchmark: update add-index-with-load doc

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Modify wording to make the document clearer and accurate

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1781

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

File: dev/benchmark/add-index-with-load.md [hunk 1]
```diff
@@ -46,9 +44,13 @@ TiKD/TiKV/PD 均使用 [TiDB Operator](https://github.com/pingcap/tidb-operator)
 | 172.31.4.172, 172.31.1.155, 172.31.9.210 |     TiKV     |
 | 172.31.7.80, 172.31.5.163, 172.31.11.123 |     TiDB     |
 
-### 测试数据
+### 使用 Sysbench 模拟线上负载
+
+使用 Sysbench 向集群导入 **1 张表，数据 200 万**。
```
Comment (yikeke) [inline dev/benchmark/add-index-with-load.md]:
```suggestion
使用 Sysbench 向集群导入 **1 张表，200 万行数据**。
```
或者 200 万条记录？

File: dev/benchmark/add-index-with-load.md [hunk 2]
```diff
@@ -79,17 +83,17 @@ sysbench $testname \
...
 ```
 
-## 测试方案 1：Add Index 目标列被频繁 Update
+## 测试方案 1：`Add Index` 目标列被频繁 Update
 
 1. 开始 `oltp_read_write` 测试。
 2. 与步骤 1 同时，使用 `alter table sbtest1 add index c_idx(c)` 添加索引。
 3. 在步骤 2 结束，即索引添加完成时，停止步骤 1 的测试。
-4. 获取指标 `alter table ... add index` 的运行时间；sysbench 在该时间段内的平均 TPS 和 QPS。
+4. 获取指标 `alter table ... add index` 的运行时间，sysbench 在该时间段内的平均 TPS 和 QPS。
```
Comment (yikeke) [inline dev/benchmark/add-index-with-load.md]:
```suggestion
4. 获取 `alter table ... add index` 的运行时间、sysbench 在该时间段内的平均 TPS 和 QPS 作为指标。
```

File: dev/benchmark/add-index-with-load.md [hunk 3]
```diff
@@ -340,5 +344,5 @@ sysbench $testname \
 
 ## 总结
 
-- 对于 `Add Index` 的目标列被频繁更新（包含 insert/delete/update）时，默认的配置下会造成较为频繁的写冲突，使得在线负载较大的影响；同时 `Add Index` 也可能由于不短地重试，需要很长的时间才能完成。在本次测试中，将 `tidb_ddl_reorg_worker_cnt` × `tidb_ddl_reorg_batch_size` 调整为默认值的 1/32（例如 `tidb_ddl_reorg_worker_cnt` = 4，`tidb_ddl_reorg_batch_size` = 256）可以得到较好的效果。
-- 对于 `Add Index` 的目标列仅涉及查询负载时，或者与线上负载不直接相关时，可以直接使用默认值。
+- 当 `Add Index` 的目标列被频繁更新（包含 `UPDATE`, `INSERT` 和 `DELETE`）时，默认配置会造成较为频繁的写冲突，使得在线负载较大；同时 `Add Index` 也可能由于不断地重试，需要很长的时间才能完成。在本次测试中，将 `tidb_ddl_reorg_worker_cnt` 和 `tidb_ddl_reorg_batch_size` 的乘积调整为默认值的 1/32（例如 `tidb_ddl_reorg_worker_cnt` = 4，`tidb_ddl_reorg_batch_size` = 256）可以取得较好的效果。
```
Comment (yikeke) [inline dev/benchmark/add-index-with-load.md]:
```suggestion
- 当 `Add Index` 的目标列被频繁更新（包含 `UPDATE`、`INSERT` 和 `DELETE`）时，默认配置会造成较为频繁的写冲突，使得在线负载较大；同时 `Add Index` 也可能由于不断地重试，需要很长的时间才能完成。在本次测试中，将 `tidb_ddl_reorg_worker_cnt` 和 `tidb_ddl_reorg_batch_size` 的乘积调整为默认值的 1/32（例如 `tidb_ddl_reorg_worker_cnt` = 4，`tidb_ddl_reorg_batch_size` = 256）可以取得较好的效果。
```

**Comment by juliezhang1112** [comment]
PTAL @yikeke

File: dev/benchmark/add-index-with-load.md (hunk 1 above)
Comment (bb7133) [reply dev/benchmark/add-index-with-load.md]: 我觉得 200 万行数据更好一些...谢谢～

**Pull request merged**